src/bundle/Serializer/Normalizer/FieldType/TimeNormalizer.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\PersonalizationClient\Serializer\Normalizer\FieldType;
+
+use DateTime;
+use eZ\Publish\Core\FieldType\Time\Value as TimeValue;
+use eZ\Publish\SPI\Exception\InvalidArgumentType;
+use eZ\Publish\SPI\FieldType\Value;
+use Ibexa\Contracts\PersonalizationClient\Serializer\Normalizer\ValueNormalizerInterface;
+
+final class TimeNormalizer implements ValueNormalizerInterface
+{
+    public function normalize(Value $value): ?string
+    {
+        if (!$value instanceof TimeValue) {
+            throw new InvalidArgumentType('$value', TimeValue::class);
+        }
+
+        $time = $value->time;
+        if (null !== $time) {
+            $dateTime = new DateTime("@{$time}");
+
+            return $dateTime->format($value->stringFormat);
```

Comment:
Can be replaced with `(string)$value` :wink:

Reply:
Fixed: https://github.com/ibexa/personalization-client/pull/14/commits/8a3ac7f5cb9e8739305f87a01ab5d8f50d9fad90